src/Infrastructure/User/Query/Projections/UserView.php

```diff
@@ -48,8 +42,34 @@ public function serialize(): array
         ];
     }
 
+    public function uuid(): UuidInterface
+    {
+        return $this->uuid;
+    }
+
+    public function email(): string
+    {
+        return (string) $this->credentials->email;
```

Comment:
This could be `return $this->credentials->email->toString();`

Reply:
The only purpose of value objects it's encapsulate and validate a vale, so casting may work always no?

Reply:
Yep, casting always will work. I just said it to maintain always the same criteria. 

https://github.com/jorge07/symfony-4-es-cqrs-boilerplate/pull/62/files#diff-3ccf08182604f30b6bfd1e2c85ec5184L20

Reply:
Make sense